### PR TITLE
Improved documentation

### LINKED ISSUE #995: Documentation improvement
The deployment and installation docs are not that beginner friendly specially for windows users. Same goes for pull request docs.

### PR DESCRIPTION
**Pull request by abhinavsagar**
I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] is from a uniquely-named feature branch and has been rebased on top of the latest master. (If I was asked to make more changes, I have made sure to rebase onto master then too)
* [x] is descriptively named and links to an issue number, i.e. `Fixes #123`

[Fixes #995 ](https://github.com/processing/p5.js-web-editor/issues/995)

### CODE REVIEW

**Comment by abhinavsagar** [comment]
@catarak Can you please suggest what needs to be done in `Conflicts` part. Also the dependencies update in `package.json` and `package-lock.json` is being automatically done. I have not done any changes manually.

**Comment by catarak** [comment]
please look at my comments from your last PR, with respect to documentation changes. if you would like to suggest changes to package/package-lock, please open a separate PR.

**Comment by abhinavsagar** [comment]
@catarak Thanks for the changes suggested, Can you please review this now?

File: developer_docs/deployment.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ You'll only need to do this if you're testing the production environment locally
 8. Since production assumes your environment variables are in the shell environment, and not in a `.env` file, you will have to run `export $(grep -v '^#' .env | xargs)` or a similar command, see this [Stack Overflow answer](https://stackoverflow.com/a/20909045/4086967).
 9. `$ npm run start:prod`
 
+NOTE- Windows users should use `copy` instead of `cp` during installation.
```
Comment (catarak) [inline developer_docs/deployment.md]: Could you change 'NOTE- ' to '_Note_: '?

**Comment by catarak** [comment]
thanks for removing package/package-lock from this PR! maybe it's better to put the note above the installation steps, rather than after, saying something like:
"_Note_: The installation steps assume you are using a Unix-like shell. If you are using Windows, you will need to use `copy` in place of `cp`".

**Comment by abhinavsagar** [comment]
@catarak I have done the required changes. Could you please review this now?

**Pull request merged**